addon/edit/matchbrackets.js

```diff
@@ -106,13 +121,15 @@
       cm.off("cursorActivity", doMatchBrackets);
     if (val) {
       cm.state.matchBrackets = typeof val == "object" ? val : {};
+      cm.state.matchBrackets.fatCursor = 
+          cm.getWrapperElement().classList.contains('cm-fat-cursor');
```

Comment:
Additionally, please call the option `afterCursor` instead of `fatCursor`, since it may also be useful in cases that don't involve a fat cursor.